pkg/resourcecreator/pod/pod_test.go

```diff
@@ -0,0 +1,30 @@
+package pod
+
+import "testing"
+
+func TestGenerateNameFromMountPath(t *testing.T) {
```

Comment:
Kunne vell gjort dette som en table driven test: https://semaphoreci.medium.com/writing-clean-and-efficient-table-driven-unit-tests-in-go-dcee64bfd987

Reply:
<img width="603" alt="Screenshot 2023-06-06 at 11 38 14" src="https://github.com/nais/naiserator/assets/968267/6762b78a-ff5b-4bc7-8861-a5f5faacb26d">